Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -347,6 +355,8 @@ def run(self):
 
 class CompileWebKit(shell.Compile, CustomFlagsMixin):
     command = ["perl", "Tools/Scripts/build-webkit", "--no-fatal-warnings", WithProperties("--%(configuration)s")]
```

Comment:
You can consider renaming `command` to `build_command`, to keep the code between EWS and build.webkit.org similar.